docs/ops/detection/ExperimentalDetectronPriorGridGenerator_6.md

```diff
@@ -0,0 +1,97 @@
+## ExperimentalDetectronPriorGridGenerator <a name="ExperimentalDetectronPriorGridGenerator"></a> {#openvino_docs_ops_detection_ExperimentalDetectronPriorGridGenerator_6}
+
+**Versioned name**: *ExperimentalDetectronPriorGridGenerator-6*
+
+**Category**: Object detection
+
+**Short description**: An operation *ExperimentalDetectronPriorGridGenerator* operation generates prior grids of specified sizes.
+
+**Attributes**:
+
+* *flatten*
+
+    * **Description**: *flatten* attribute specifies whether the output tensor should be 2D or 4D.
+    * **Range of values**:
+      * `true` - the output tensor should be 2D tensor
+      * `false` - the output tensor should be 4D tensor
+    * **Type**: boolean
+    * **Default value**: true
+    * **Required**: *yes*
+
+* *h* (*w*)
+
+    * **Description**: *h* (*w*) attribute specifies number of cells of the generated grid with respect to height (width).
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: 0
+    * **Required**: *yes*
+
+* *stride_x* (*stride_y*)
+
+    * **Description**: *stride_x* (*stride_y*) attribute specifies the step of generated grid with respect to x (y) coordinate.
+    * **Range of values**: non-negative float number
+    * **Type**: float
+    * **Default value**: 0.0
+    * **Required**: *yes*
+
+**Inputs**
+
+* **1**: A tensor of type *T* with priors. Rank must be equal to 2 and The last dimension must be equal to 4: `[number_of_priors, 4]`. **Required.**
+
+* **2**: A 4D tensor of type *T* with input feature map. **Required.**
+
+* **3**: A 4D tensor of type *T* with input data. The number of channels of both feature map and input data tensors must match. **Required.**
+
+**Outputs**
+
+* **1**: A tensor of type *T* with priors grid with shape `[featmap_height * featmap_width * number_of_priors, 4]` if flatten is `true` or `[featmap_height, featmap_width, number_of_priors, 4]` otherwise, where `featmap_height` and `featmap_width` are spatial dimensions values from second input.
+
+**Types**
+
+* *T*: any supported floating point type.
+
+**Detailed description**: 
+
+Operation computes prior grids by following:
+
+    for (int ih = 0; ih < featmap_height; ++ih)
+        for (int iw = 0; iw < featmap_width; ++iw)
+            for (int s = 0; s < number_of_priors; ++s)
+                data[0] = priors[4 * s + 0] + stride_x * (iw + 0.5)
+                data[1] = priors[4 * s + 1] + stride_y * (ih + 0.5)
+                data[2] = priors[4 * s + 2] + stride_x * (iw + 0.5)
+                data[3] = priors[4 * s + 3] + stride_y * (ih + 0.5)
+                data += 4
```

Comment:
Maybe these lines should be formatted as C++ code?

Reply:
Updated, but this part is still uses pseudocode-like style. Do you think C++ formatting will be better to understanding?

Reply:
If we write this C++-code, then we should use Markdown tags to syntax highlight, that is
```c++
    for (int ih = 0; ih < layer_height; ++ih)
        for (int iw = 0; iw < layer_width; ++iw)
            for (int s = 0; s < number_of_priors; ++s)
                output_data[0] = priors[4 * s + 0] + step_w * (iw + 0.5)
                output_data[1] = priors[4 * s + 1] + step_h * (ih + 0.5)
                output_data[2] = priors[4 * s + 2] + step_w * (iw + 0.5)
                output_data[3] = priors[4 * s + 3] + step_h * (ih + 0.5)
                output_data += 4
```
But we can replace this code with some simple description, as @lazarevevgeny requested

Reply:
Done.